src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -195,7 +195,7 @@ class UserService(protected val userInfo: UserInfo, dataSource: SlickDataSource,
     getUserStatus(RawlsUserRef(RawlsUserSubjectId(userInfo.userSubjectId)))
   }
 
-  def getUserStatus(userRef: RawlsUserRef): Future[PerRequestMessage] = withUser(userRef) { user =>
+  def getUserStatus(userRef: RawlsUserRef): Future[PerRequestMessage] = withUserInTransaction(userRef) { user =>
```

Comment:
In this case (and the 2 following) we only need the user and only need the transaction for that time. The rest of the method does not need a transaction. This may be been left over from orient days where we had inFutureTransaction vs inTransaction.

It is probably not a big deal to hold the transaction for the extent but then you probably need to change the other Futures to DBIOActions (DBIO.from(...))

Alternately just inline withUserInTransaction